builder.md

```diff
@@ -100,20 +111,22 @@ A builder MUST provide the following variables to template files:
 - `scheme-author` - obtained from the `author` key of the scheme file
 - `scheme-description` - obtained from the `description` key of the scheme file (fallback value: `scheme-name`)
 - `scheme-slug` - the scheme filename made lowercase, not including the `.yaml` extension
-- `base00-hex` to `base0F-hex` - 6-digit hex color value obtained from the scheme file. MUST NOT include a leading `#`. e.g "7cafc2".
-- `base00-hex-bgr` to `base0F-hex-bgr` - built from a reversed version of all the hex values e.g "c2af7c"
-- `base00-hex-r` to `base0F-hex-r` - red component of the hex color value. e.g "7c"
-- `base00-hex-g` to `base0F-hex-g` - green component of the hex color value. e.g "af"
-- `base00-hex-b` to `base0F-hex-b` - blue component of the hex color value. e.g "c2"
-- `base00-rgb-r` to `base0F-rgb-r` - red component as a value between `0` and `255`. e.g "124"
-- `base00-rgb-g` to `base0F-rgb-g` - green component as a value between `0` and `255`. e.g "175"
-- `base00-rgb-b` to `base0F-rgb-b` - blue component as a value between `0` and `255` e.g "194"
-- `base00-dec-r` to `base0F-dec-r` - red component as a value between `0` and `1.0`. e.g "0.4863"
-- `base00-dec-g` to `base0F-dec-g` - green component as a value between `0` and `1.0`. e.g "0.6863"
-- `base00-dec-b` to `base0F-dec-b` - blue component as a value between `0` and `1.0`. e.g "0.7608"
+- `scheme-style` - obtained from the `style` key of the scheme file (fallback value: "base16")
+
+Additionally, a builder MUST provide the following variables for each defined color:
+
+- `{{ color-name }}-hex` - 6-digit hex color value obtained from the scheme file. MUST NOT include a leading `#`. e.g "7cafc2".
```

Comment:
The idea here is an improvement but I think we need something more generic... I call them `slots`... and even in Base17 (with only simple semantics) they aren't "color names" anymore... `diff_inserted-hex` isn't a "color name"...

Again this following my thinking that we should NOT be evolving base16 further - so the only way I can understand these changes is in light of the first new style system we create...  this would have been a nice improvement to the old base16 spec, but that's not what we're doing anymore.